content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst

```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
+.. image:: one_step_manufacturing/work-order-card.png
    :align: center
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step, and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
 
-After opening the tablet view, Odoo *Manufacturing* automatically starts a timer that keeps track of
-how long the work order takes to complete. After completing the work order, click the
-:guilabel:`Mark as Done` button in the top-right corner of the tablet view.
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is marked as
+completed automatically, without a pop-up window appearing.
 
-Clicking :guilabel:`Mark as Done` while there is at least one more work order left to complete opens
-a page that lists the next work order. Click on that work order to open it in the tablet view.
+The final step on a work order card is titled *Register Production*. This step is used to register
+the number of product units that were produced. If the number produced is equal to the number that
+the |MO| was created for, click the :guilabel:`# Units` button on the right side of the line to
+automatically register that number as the quantity produced.
 
-Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
-button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
-:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done` and register the
-manufactured product(s) into inventory.
+If a different number must be entered, click the :guilabel:`Register Production` step to open a
+pop-up window. Enter the number of units produced in the :guilabel:`Units` field, and then click
+:guilabel:`Validate` to register that number.
 
-It is also possible to complete the final operation while keeping the |MO| open, by clicking
-:guilabel:`Mark as Done`. In this case, the |MO| can be closed later by clicking the
-:guilabel:`Produce All` button on the order.
+.. note::
+   The *Register Production* step appears on every work order card. It must be completed for the
+   first work order that is processed. After doing so, the step appears as already completed for
+   each remaining work order in the |MO|.
+
+After completing all of the steps for a work order, a button appears on the footer of the work order
+card. If any other work orders must be completed before the |MO| can be closed, the button is titled
+*Mark as Done*. If there are no additional work orders to complete, the button is titled *Close
+Production*.
```

Comment:
I'm not sure when we use italics, but my known example is italicizing terms we're defining for the first time, like [specified in this doc](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.90nibixh8llr)

If you're talking bout the text, "Mark as Done," maybe surround it with quotes? but i don't see why *not* to use a guilabel here to refer to the button itself, directly. What do you think? :smile: 
```suggestion
card. The :guilabel:`Mark as Done` button appears when any other work orders must be completed to
close the |MO|. If there are no additional work orders to complete, the :guilabel:`Close Production`
button appears, instead.
```

Also, optional rewording suggestion to use more active voice! feel free to ignore